syntaxnet/ud_train/convert.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+#-*- coding: utf8 -*-
+
+import os
+import sys
+from   optparse import OptionParser
```

Comment:
remove extra spaces. I am wonder if it's better to use the tf flag parsing instead.